hack/install-deps.sh

```diff
@@ -140,12 +139,6 @@ fi
 checkout_repo ${CONTAINERD_PKG} ${CONTAINERD_VERSION} ${CONTAINERD_REPO}
 cd ${GOPATH}/src/${CONTAINERD_PKG}
 if ${COOK_CONTAINERD}; then
-  # Verify that vendor.conf is in sync with containerd before cook containerd,
```

Comment:
We do still need this for now.

Reply:
I understand the desire to have a tool for testing whether they are in sync. Don't see why they have to be in 100% sync before allowing a PR though. Why hard requirement?

Reply:
@mikebrow The problem is that in following code, we copy cri-containerd vendor to containerd vendor directory directly.

If there is a go file in older version in containerd, but no such file in newer version in cri-containerd. This file will not be overwritten during the move, and containerd may not build because of this extra go file.

If we don't want to enforce, we need to at least print a warning here, so that if it doesn't build we know it might be caused by vendor not match.

Reply:
ok... updating the PR to put it back in.. modifying the todo a bit to cover this part as well. Cheers.